src/chrome/content/rules/DiscoveryPlace.xml

```diff
@@ -1,21 +1,22 @@
 
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://discoveryplacekids.org/ => https://www.discoveryplacekids.org/: (51, "SSL: no alternative certificate subject name matches target host name 'www.discoveryplacekids.org'")
-Fetch error: http://www.discoveryplacekids.org/ => https://www.discoveryplacekids.org/: (51, "SSL: no alternative certificate subject name matches target host name 'www.discoveryplacekids.org'")
+	Login required:
+		email.discoveryplace.org
+
+	Invalid certificate:
+		online.discoveryplace.org
+		www.science.discoveryplace.org
+		*.discoveryplacekids.org
```

Comment:
I could not find any subdomain with a valid certificate for the domain `discoveryplacekids.org` (note that this is a domain).